apispec/schema.yml

```diff
@@ -423,6 +423,93 @@ paths:
                     type: string
                     format: date
 
+  /transactions/{transaction_id}/insolvency/progress-report:
+    post:
+      tags:
+        - "progress report"
```

Comment:
Tags should be capitalised for consistency i.e. "Progress Report". Same for both other instances below

Reply:
fixed